src/test/ui/thread-local-issue-37508.rs

```diff
@@ -0,0 +1,36 @@
+// only-x86_64
```

Comment:
As mentioned in https://github.com/rust-lang/rust/issues/73494#issue-641682406, we generally avoid adding tests to the root, I think this is related to the panic operations (right?) so `panics` should be better.

Reply:
If I understand the issue correctly, it is more related to thread local. I created a new directory `src/test/ui/thread-local` and put the test in there.